### ISSUE TITLE
Youtube links giving "Invalid URL"

### ISSUE THREAD

**Issue by ModerateWinGuy** [MerlinVR/USharpVideo#5]
As of yesterday whenever I try to play a youtube video I've been getting "Invalid URL"
Has a VRC update broken the player somehow?

**Comment by MerlinVR** [comment]
YouTube likely changed how their links get resolved as they do every few months. I can't do anything to resolve the Invalid URL error. Usually this would mean VRC needs to update YouTubeDL since that is what they use to resolve URLs, but YTDL's main repository is knocked down for the moment. https://arstechnica.com/tech-policy/2020/10/github-boots-popular-youtube-download-tool-after-riaa-claim/ So it may take longer for YouTubeDL to issue an updated build, which they can still do through their website which has not been taken down.

tl;dr I can't fix this, VRC needs to, but they need the tool they use to be updated to fix it which may take longer than usual.

I'll leave this issue open for now so other people running into the same errors can see this.

**Comment by MerlinVR** [comment]
Looks like YouTubeDL published an updated build today so VRC should just need to update to it on their side.

**Comment by ModerateWinGuy** [comment]
Cool, thanks for the updates!

**Comment by ModerateWinGuy** [comment]
Seems to be working fine now so I'll close the issue.
